web-app/uni-todo-app/src/components/TodoInputField.tsx

```diff
@@ -2,13 +2,20 @@ import { useCallback, useState } from "react";
 import { ITodoEntity } from "../store/interfaces/ITodoEntity";
 import { insertTodoEntity } from "../store/asyncActions/todoAsyncActions";
 import { useAppDispatch } from "../hooks";
+import { AlertWindow } from "./AlertWindow";
 
 export const TodoInputField = () => {
   const [input, setInput] = useState("");
+  const [error, setError] = useState("");
 
   const dispatch = useAppDispatch();
 
   const addTodoEntity = useCallback(() => {
+    if(!input.trim()) {
```

Comment:
You want to show the error only when the input value is empty. So better use `!!input.trim()`

Check this out https://webionista.com/javascript-double-exclamation-mark/#:~:text=In%20short%2C%20The%20double%20exclamation,for%20use%20in%20conditional%20statements.

Reply:
Yeah I'm a bit confused on that. When i use !! instead, an empty value is accepted. If the string is empty then from what i understand !!input.trim() is false, so if the condition is false the block inside will not run

Reply:
My bad! Long day :P